pkg/server/server.go

```diff
@@ -15,26 +16,26 @@ import (
 )
 
 type Server struct {
-	Datasource Datasource
+	Datasource    Datasource
+	channelPrefix string
+	closeCh       chan struct{}
+	queryMux      *datasource.QueryTypeMux
```

Comment:
rather than build the map on every request, this saves builds it once on startup